lib/components/HeaderMenu.js

```diff
@@ -0,0 +1,110 @@
+import React, { Component, Fragment } from 'react';
+import {
+  MenuItem,
+  Popper,
+  MenuList,
+  IconButton,
+  Fade,
+  Paper,
+  ClickAwayListener,
+} from '@material-ui/core';
+import { Menu as MenuIcon, Close as CloseIcon } from '@material-ui/icons';
+import Link from './Link';
+
+class HeaderMenu extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      anchorEl: null,
+    };
+  }
+
+  handleMenuToggle = event => {
+    const { anchorEl } = this.state;
+    this.setState({
+      anchorEl: anchorEl === null ? event.currentTarget : null,
+    });
+  };
+
+  handleMenuClose = () => {
+    this.setState({
+      anchorEl: null,
+    });
+  };
+
+  handleListKeyDown = event => {
+    if (event.key === 'Tab') {
+      event.preventDefault();
+      this.setState({
+        anchorEl: null,
+      });
+    }
+  };
+
+  render() {
+    const { anchorEl } = this.state;
+    const { items, placement } = this.props;
+    const isMenuOpen = Boolean(anchorEl);
+
+    return (
+      <Fragment>
+        <IconButton
+          size="medium"
+          color="inherit"
+          aria-label="open header menu"
+          aria-haspopup="true"
+          onClick={this.handleMenuToggle}
+        >
+          {isMenuOpen ? <CloseIcon /> : <MenuIcon />}
+        </IconButton>
+
+        <Popper
+          open={isMenuOpen}
+          anchorEl={anchorEl}
+          role={undefined}
+          transition
+          disablePortal
+          placement={placement || 'bottom-start'}
+        >
+          {({ TransitionProps }) => (
+            <Fade {...TransitionProps}>
+              <Paper>
+                <ClickAwayListener onClickAway={this.handleMenuClose}>
+                  <MenuList
+                    id="menu-list-grow"
```

Comment:
Looks like this `id` is not being referenced anywhere. Could we remove it?

Reply:
Removed redundant id - all done.